tests/conftest.py

```diff
@@ -608,3 +611,37 @@ async def setup_sim():
     sim_client = SimClient(sim)
     await sim.farm_block()
     return sim, sim_client
+
+
+@pytest.fixture(scope="function")
+def tmp_chia_root(tmp_path):
+    """
+    Create a temp directory and populate it with an empty chia_root directory.
+    """
+    path: Path = tmp_path / "chia_root"
+    path.mkdir(parents=True, exist_ok=True)
+    return path
+
+
+@pytest.fixture(scope="function")
+def root_path_populated_with_config(tmp_chia_root) -> Path:
+    """
+    Create a temp chia_root directory and populate it with a default config.yaml.
+    Returns the chia_root path.
+    """
+    root_path: Path = tmp_chia_root
+    create_default_chia_config(root_path)
+    return root_path
+
+
+@pytest.fixture(scope="function")
+def root_path_and_config_with_address_prefix(
+    root_path_populated_with_config: Path, prefix: str
+) -> Tuple[Path, Dict[str, Any]]:
+    updated_config: Dict[str, Any] = {}
+    with lock_and_load_config(root_path_populated_with_config, "config.yaml") as config:
+        if prefix is not None:
+            config["network_overrides"]["config"][config["selected_network"]]["address_prefix"] = prefix
+            save_config(root_path_populated_with_config, "config.yaml", config)
+        updated_config = config
+    return root_path_populated_with_config, updated_config
```

Comment:
This can just yield the config.  If the test also needs the root path they can just depend on the `tmp_chia_root` directly.  This avoids having to unpack in the test as well as avoiding tuple return values.

Reply:
Renamed and updated so that the fixture yields the config.